test/acceptance/helpers/dummy.js

```diff
@@ -0,0 +1,50 @@
+import express from 'express'
+import basicAuth from 'express-basic-auth'
+import path from 'path'
+
+/**
+ * Creates an express server to serve a static dummy app with optional basic
+ * authentication support.
+ *
+ * @param {object} [options={}] - Options used in creating the express server
+ * @param {object} [options.auth] - Pairs of username and passwords
+ * @returns {object} An object containing app and server references
+ */
+function createDummyApp({ auth } = {}) {
+  let app = express()
+  let server = app.listen(9999)
+  let staticMiddleware = express.static(path.join(__dirname, '../dummy'))
+
+  app.use(staticMiddleware)
+
+  if (auth != null) {
+    let authMiddleware = basicAuth({ challenge: true, users: auth })
+    app.use('/protected', authMiddleware, staticMiddleware)
+  }
+
+  return { app, server }
+}
+
+/**
+ * Hooks into Mocha's `beforeEach` and `afterEach` to create and close the dummy
+ * server before and after each test. A local context is mutated and cleaned up
+ * to handle memory leaks and returning helpers for use in tests.
+ *
+ * @returns {object} An object containing app and server references
+ */
+export default function setupDummyApp(options) {
```

Comment:
Not sure what you mean by 'embed' in this instance. Like embed the netifly site itself? Or just move those pages over into this dummy app? 

My concern with the former would be that you need to make and merge changes to an entirely different repo just to write a test. With this dummy app, I'd like to make sure all tests stay within this repo. So we don't hit _any_ live site. Not even the test site (cause that's still a live site that would cause our tests to fail if any internet issues arise).